Dockerfile

```diff
@@ -47,6 +43,10 @@ ENV SYMFONY_ENV prod
 # Fix AllowOverride All for /var/www/html
 COPY docker/apache2.conf /etc/apache2/apache2.conf
 
+# Copy in our custom php.inis
+COPY docker/php.ini /etc/php5/apache2/php.ini
+COPY docker/php-cli.ini /etc/php5/cli/php.ini
```

Comment:
I stedet for a kopiere hele ini-filer ind over de eksisterende er det så ikke bedre at have ini-filer med kun den justerede memory_limit og lægge den i `/etc/php/5.6/apache2/conf.d/` og `/etc/php/5.6/cli/conf.d/` eller lignende?

Reply:
Jow, det er en bedre idé. Consider it done!